main.go

```diff
@@ -78,41 +78,34 @@ func (c *Collector) extractMetrics(ch chan<- prometheus.Metric, rm resourceMeta,
 		}
 		metricName = invalidMetricChars.ReplaceAllString(metricName, "_")
 
+		val := float64(0)
```

Comment:
Maybe better to write `var val float64` as more convenient in this file?

https://github.com/percona/azure_metrics_exporter/blob/master/main.go#L131
https://github.com/percona/azure_metrics_exporter/blob/master/main.go#L222-L223
etc.

Reply:
Yes, true. Changed.